SPCTR-3406 [Container] Padding should inherit from desktop (#3524)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Improvement: Container - Updated padding settings so responsive devices takes default value from desktop css.

Reviewer: Let's have something like this?

```suggestion
* Improvement: Container - Updated responsive padding settings to inherit from bigger devices by default.
```

Author: updated
